packages/components/src/logos/facebook-logo/index.tsx

```diff
@@ -0,0 +1,67 @@
+const FacebookLogo = ( props: React.SVGProps< SVGSVGElement > ) => (
+	<svg
+		width="78"
+		height="27"
+		viewBox="0 0 78 27"
+		fill="none"
+		xmlns="http://www.w3.org/2000/svg"
+		{ ...props }
+	>
+		<title>WordPress VIP client logo for Facebook</title>
+		<mask id="mask0_233_14595" maskUnits="userSpaceOnUse" x="0" y="10" width="77" height="9">
+			<path d="M76.9837 10H0.5V18.1053H76.9837V10Z" fill="white" />
+		</mask>
+		<g mask="url(#mask0_233_14595)">
+			<path
+				fill-rule="evenodd"
```

Comment:
This is throwing a warning in the console. I think this should be `fillRule`. also for a few other cases below